docs/unified/msi/firmware-update.md

```diff
@@ -14,22 +14,39 @@ Alternatively, it can be checked in the `BIOS Setup Menu`.
 
 ## Prerequisites
 
-Before proceeding, ensure that the firmware protections are disabled in
-[Dasharo Security Options](../../dasharo-menu-docs/dasharo-system-features.md).
-Both `BIOS boot medium lock` and `Enable SMM BIOS write protection` should be
-unchecked. [UEFI Secure Boot](../../dasharo-menu-docs/device-manager.md#secure-boot-configuration)
-must be disabled as well (uncheck `Attempt Secure Boot` if
-`Current Secure Boot State` does not say `Disabled`). To apply changes you will
-need to reboot.
+Depending on firmware version (1) there may be manual steps required to ensure that
+the firmware can be updated.
+{ .annotate }
+
+1. v0.9.0 for Z790-P and v1.1.2 for Z690-A introduced support for Firmware
+   Update Mode
+
+=== "Firmware Update Mode"
+    Recent Dasharo releases come with Firmware Update Mode, an option in the
+    setup menu that disables all firmware protections for the duration of one
+    boot, during which firmware can be updated. To enter Firmware Update Mode,
+    open the [Dasharo Security Options](../../dasharo-menu-docs/dasharo-system-features.md)
+    menu and select the `Firmware Update Mode` option.
```

Comment:
This should be rather moved to a generic place where we could link to from any board that support this option.

I would avoid doing the tabs for such purpose

Reply:
@miczyg1 Do you have any specific location in mind?

Reply:
We can go either with `docs/guides/firmware-update.md` or `docs/osf-trivia-list/firmware-update.md` (this is where generic flashrom documentation went), where we could consolidate some generic procedure how to perform firmware update, including enabling FUM.

Reply:
Moved to `docs/guides/firmware-update.md` as suggested in commit https://github.com/Dasharo/docs/pull/644/commits/d7fe7f67e6c923c1f237ef2867d59a996aecf74b